### PR TITLE
Format remainder of VB code base

### PR DESCRIPTION
**Pull request by jaredpar**
Now that the comment formatting issue is fixed in the Formatter type, we can run the code formatter on the remainder of the VB code base.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @AlekseyTs, @gafter, @agocke, @VSadov, @VladimirReshetnikov, @jasonmalinowski

File: src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb [hunk 1]
```diff
@@ -4144,27 +4144,27 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
...
         Private Shared s_isFunctionWithoutArguments As Func(Of Symbol, Boolean) = Function(sym)
-                                                                                    If sym.Kind = SymbolKind.Method Then
-                                                                                        Dim method = DirectCast(sym, MethodSymbol)
-                                                                                        Return Not method.IsSub() AndAlso
-                                                                                               Not method.IsGenericMethod AndAlso
-                                                                                               method.CanBeCalledWithNoParameters
-                                                                                    End If
-                                                                                    Return False
-                                                                                End Function
+                                                                                      If sym.Kind = SymbolKind.Method Then
```
Comment (gafter) [inline src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb]: Both the old and the new formatting are terrible (too deeply indented), but that should be handled separately.

File: src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/FlowTestBase.vb [hunk 2]
```diff
@@ -140,7 +140,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
         End Function
 
         Private Sub AdjustToken(ByRef token As SyntaxToken)
-tryAgain :
+tryAgain:
```
Comment (gafter) [inline src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/FlowTestBase.vb]: Are labels really supposed to be at the left margin?

Comment (jaredpar) [inline src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/FlowTestBase.vb]: CC @basoundr

File: src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb (hunk 1 above)
Comment (jaredpar) [inline src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb]: I agree.  If we hit insert a line break after the `=` in `= Function it should fix the indent nicely.

File: src/Compilers/VisualBasic/Test/Syntax/Syntax/SyntaxRewriterTests.vb [hunk 3]
```diff
@@ -543,139 +543,139 @@ End Class
 #Region "Helper Types"
 
         Private Sub TestGreen(input As String, output As String, rewriter As GreenRewriter, isStmt As Boolean)
-                Dim red As VisualBasicSyntaxNode
-                If isStmt Then
-                    red = SyntaxFactory.ParseExecutableStatement(input)
-                Else
-                    red = SyntaxFactory.ParseCompilationUnit(input)
-                End If
+            Dim red As VisualBasicSyntaxNode
```
Comment (gafter) [inline src/Compilers/VisualBasic/Test/Syntax/Syntax/SyntaxRewriterTests.vb]: You confused github's differ!

**Comment by gafter** [comment]
Everything looked good to me as far as I was able to stay awake.

File: src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb (hunk 1 above)
Comment (tmat) [inline src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb]: +1 for moving lambda on the next line.

Comment (AdamSpeight2008) [inline src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb]: @gafter I agree, the indenting really needs to 2 spaces rather than 4.

Comment (jaredpar) [inline src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb]: @AdamSpeight2008 we've been over this.  The spacing requirement for dotnet Foundation projects is 4 spaces.

File: src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/FlowTestBase.vb (hunk 2 above)
Comment (basoundr) [inline src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/FlowTestBase.vb]: @gafter @jaredpar Yes, Labels are always aligned to (Column 0)Left margin in VB. Also no spacing between the label and the colon. The resultant above is the desired format.

**Pull request closed without merging**